contracts/DNSSECImpl.sol

```diff
@@ -63,7 +63,7 @@ contract DNSSECImpl is DNSSEC, Owned {
         anchors = _anchors;
         rrsets[keccak256(hex"00")][DNSTYPE_DS] = RRSet({
             inception: uint32(0),
-            inserted: uint64(block.timestamp),
+            expiration: uint32(3767581600), // May 22 2089
```

Comment:
What is the meaning of this specific date 2089?

Reply:
👍